ci-operator/jobs/openshift/installer/openshift-installer-release-4.5-presubmits.yaml

```diff
@@ -989,6 +989,85 @@ presubmits:
         secret:
           secretName: regcred
     trigger: (?m)^/test( | .* )e2e-openstack,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - release-4.5
+    cluster: api.ci
+    context: ci/prow/e2e-ovirt
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-installer-release-4.5-e2e-ovirt
+    optional: true
+    rerun_command: /test e2e-ovirt
+    run_if_changed: ^([^d]|d(d|o(d|cd))*([^do]|o([^cd]|c[^ds])))*(d(d|o(d|cd))*(oc?)?)?$
```

Comment:
We should start leveraging run_if_changed more effectively in these jobs. Don't need to do that now though.